packages/mui-joy/src/GlobalStyles/index.tsx

```diff
@@ -1,11 +1 @@
-import * as React from 'react';
```

Comment:
@mj12albert the index file also needs `'use client';` right?

Reply:
Ah, based on https://github.com/mui/material-ui/pull/38967/files#diff-dafb0dc18749ba2c9f0efdec79126704f72fc6a105b2247580eb3d268c067d84, it is not needed.